### DISCUSSION TITLE
ISAC Simulation

### DISCUSSION THREAD

**Discussion by fanyimeng0** [NVlabs/sionna#375] [General]
Hi, have you considered conducting ISAC-related simulations using Sionna? Based on my assumptions, for simple downlink simulations,  simulations can be done by computing paths through Sionna RT. Have you considered other tools to support more complex simulations?

**Comment by jhoydis** [comment]
Hi @fanyimeng0,

Could you be a little bit more specific? I am not sure what other tools or scenarios you are referring to.

**Reply by fanyimeng0** [reply]
Thank you for your response. I would like to know 1. Could Sionna provide system-level simulation, assuming I provide the locations of all transmitters？  2. Can I simulate the dynamic scenario in Sionna. For example, assuming a car is moving at a speed of 1 m/s, can I calculate the channel every second? Changing the position in Blender seems works, but sometimes it might be too complex.
